tests/mochainon.d.ts

```diff
@@ -1,12 +1,13 @@
-declare module 'mochainon';
+declare module 'mochainon' {
 
-import * as chai from 'chai';
-import * as sinon from 'sinon';
+	import * as chaiImport from 'chai';
```

Comment:
Following the references, chai has a top-level `declare const chai: ...`, which I think actually declares that as a global type, so maybe you don't even need this. Presumably that's because it gets injected globally in a lot of normal chai usage? Feels weird though, I think we should keep importing it explicitly.